### DISCUSSION TITLE
Windows server :(

### DISCUSSION THREAD

**Discussion by esisl** [cockpit-project/cockpit#18658] [Q&A] [answered]
Good day!

Can you tell me how to update the cockpit if it is deployed on a windows server?

**Comment by KKoukiou** [comment]
Cockpit is not replayed on a windows server, you just use the browser from your windows PC in order to access remotely cockpit which is installed on a linux server. You can use 'Software updates' cockpit page to update cockpit itself.

**Comment by esisl** [comment]
But it used to work under Windows, right?
And now absolutely nothing?
:(

**Reply by martinpitt** [reply]
No, cockpit has never run on anything else than Linux (with systemd).

**Comment by esisl** [comment]
ubuntu1~14.04.3
won't work either?

**Reply by martinpitt** [reply]
No -- Please don't put an Ubuntu 14.04 machine onto the public internet! That has been EOL for 4 years now.

**Comment by esisl** [comment, marked as answer]
Sorry. I'm crazy


I don't need a cockpit, but a CMS Cockpit!!!
